### PR TITLE
Add test to ensure LogStorage secrets controller takes no action on m…

### PR DESCRIPTION
**Pull request by Josh-Tigera**
…anaged clusters

Also includes bonus test args consolidation.

## For PR author

- [x] Tests for change.
~~- [ ] If changing pkg/apis/, run `make gen-files`~~
~~- [ ] If changing versions, run `make gen-versions`~~

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

File: pkg/controller/logstorage/secrets/secret_controller_test.go [hunk 1]
```diff
@@ -476,6 +476,30 @@ var _ = Describe("LogStorage Secrets controller", func() {
...
 	})
+
+	It("should not take any action in managed cluster", func() {
+		// Create a LogStorage instance with a default configuration.
+		ls := &operatorv1.LogStorage{}
+		ls.Name = "tigera-secure"
+		ls.Status.State = operatorv1.TigeraStatusReady
+		CreateLogStorage(cli, ls)
+
+		// Create a ManagementCluster object
```
Comment (caseydavenport) [inline pkg/controller/logstorage/secrets/secret_controller_test.go:487]: typo: ManagementClusterConnection

Comment (Josh-Tigera) [reply pkg/controller/logstorage/secrets/secret_controller_test.go:487]: Fixed.

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Comment by marvin-tigera** [comment]
Removing "merge-when-ready" label due to new commits

**Pull request merged**